routers/api/packages/pypi/pypi.go

```diff
@@ -177,3 +177,7 @@ func UploadPackageFile(ctx *context.Context) {
 
 	ctx.Status(http.StatusCreated)
 }
+
+func hasInvalidMetadata(packageName, packageVersion string) bool {
```

Comment:
I would prefer a "positive" method like `isValidNameAndVersion`. What do you think?

Reply:
Agree it is easier to read/interpret, made an update.